### DISCUSSION TITLE
muduo::net::Buffer中有unread方法吗

### DISCUSSION THREAD

**Discussion by 294068860** [chenshuo/muduo#663] [General]
muduo::net::Buffer中有unread方法吗，我找了好几个版本都没有，但是同事以前写的代码明明调用了这个函数，是怎么回事啊
buff->unread

**Comment by ggandycong** [comment]
刘德聪，已经收到你的邮件，请你放心
祝你生活愉快，笑口常开

**Comment by chenshuo** [comment]
> 刘德聪，已经收到你的邮件，请你放心 祝你生活愉快，笑口常开

Please don't spam.

**Comment by chenshuo** [comment]
> muduo::net::Buffer中有unread方法吗，我找了好几个版本都没有，但是同事以前写的代码明明调用了这个函数，是怎么回事啊 buff->unread

Buffer::prepend() ?
